src/utils/env.ts

```diff
@@ -46,20 +49,28 @@ const spacesValidator = makeValidator<string>(validateSpaces);
  * validateEmailTo("foo@bar.com"); // valid
  * validateEmailTo("foo+foo@bar.com"); // valid
  * validateEmailTo("foo@bar.com,bar@foo.com"); // valid
+ * validateEmailTo("foo.eth:foo@example.com"); // valid
+ * validateEmailTo("foo.eth:foo@example.com,bar.eth:bar@example.com"); // valid
  * validateEmailTo("foo"); // invalid
  * validateEmailTo("foo@"); // invalid
  * validateEmailTo("@foo"); // invalid
  * validateEmailTo("foo@bar.com,"); // invalid
  * validateEmailTo("foo@bar.com,,"); // invalid
+ * validateEmailTo("foo.eth:"); // invalid
  */
 export const validateEmailTo = (input: string) => {
   // Validate one email address following the validation applied to <input type="email">
   // https://html.spec.whatwg.org/multipage/input.html#email-state-(type=email)
   const oneEmailRegexText =
-    "[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*";
+    "[a-zA-Z0-9\\.!#$%&'*+\\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*";
```

Comment:
This change fixed a bug introduced by simply copypasting the regex into a string to build it. The `\` characters were removed as the prettifier interpreted that it wasn't required to escape the next character IN A STRING (while it is valuable in a RegExp). This resulted in wrong inputs being allowed.

Reply:
@wadader Not sure I get what clarifications are you requesting regarding those lines. The buggy line was the unescaped long string. The rest of the code was added to support a change in the expected input format, and it is building a regex by part. As the regex sections are reused, I thought it would be easier to read and validate by having those sections in named strings variables.

Reply:
Each variable is a step in the regex build. Got it